wd_helper.au3

```diff
@@ -1061,7 +1061,7 @@ Func _WD_ElementSelectAction($sSession, $sSelectElement, $sCommand)
 		__WD_ConsoleWrite($sFuncName & ': ' & ((IsArray($vResult)) ? "(array)" : $vResult))
 	EndIf
 
-	Return SetError(__WD_Error($sFuncName, $iErr), $_WD_HTTPRESULT, $vResult)
+	Return SetError(__WD_Error($sFuncName, $iErr, Default, $iExt), $_WD_HTTPRESULT, $vResult)
```

Comment:
undeclared variable

Reply:
Fixed in: [_WD_ElementSelectAction $iExt fix](https://github.com/Danp2/au3WebDriver/pull/244/commits/1cacedb02289a53fb30b34956dec36b16d0963fb)
This was because I'm using version which already have `_WD_ElementSelectAction - multiple selection support PR#212`